### DISCUSSION TITLE
Looking for guideline on Support model for Migration of Greenlplum to Cloudberry

### DISCUSSION THREAD

**Discussion by danish7387** [apache/cloudberry#759] [Q&A]
Hi Team, 
Hi Support,
This side Danish Hussain, at our company Afiniti, we planning to move Greenplum database to Cloudberry, therefore like to have a call with you guys to understand the support model and legal aspects as follows

1-Support Model

We need to confirm whether Cloudberry provides binaries along with technical support or if  offering is limited to technical support only. Our requirement is to identify a vendor capable of delivering comprehensive enterprise support, which includes both technical support and binaries. Based on current insights, this level of support may only be available through Cloudberry,if you can guide to reach a right vendor for the support.

2-Legal Aspect

We also seeking clarification regarding the legal aspects of Cloudberry's incubation under the Apache Software Foundation. Specifically, could you confirm whether Cloudberry is a purely U.S.-originated product moving forward? Additionally, what measures, if any, are in place to mitigate potential external influences, particularly from non-U.S. entities, in the project's development or governance?

As our major clients are U.S.-based, understanding the extent of any potential non-U.S. influence, particularly from Chinese entities, is crucial for us to evaluate the project's alignment with our compliance and operational requirements.
Your guidance on this matter will be highly valuable for our decision-making process

**Comment by 2shar-p** [comment]
Danish Hussain... 

Welcome to the Apache Cloudberry Community!
As you join the Apache Cloudberry project, it’s important to understand the distinctions between projects and products within the context of the Apache Software Foundation (ASF). Here's an overview to set expectations and guide your journey.

What is Apache Cloudberry?

Apache Cloudberry is an open-source project under the ASF, focused on producing freely available source code. 

Key highlights:
The project will produce source code releases and may optionally provide deployable binaries (binary convenience artifacts).
Any binaries released are offered without any warranty and are not intended to meet the needs of customers requiring enterprise-grade support.

Neutral Development: 
Neither the source code nor binaries are tied to a specific country. Apache Cloudberry is a collaborative global project under the ASF, a U.S.-based non-profit organization.

Independent from Influence: ASF projects operate free from corporate or government control, ensuring neutrality and transparency.
Products and Commercial Support

While Apache Cloudberry is an ASF project, commercial products based on its technology are developed and supported by independent companies worldwide.

Notably, U.S.-based companies like SynxData.com and Enfuse.IO offer global support for Cloudberry-based solutions. These organizations specialize in providing:

Professional-grade enterprise support tailored to the specific needs of businesses.
Enhanced features and proprietary components to complement the open-source foundation.
Global reach, ensuring support for organizations operating in any region.

Exclusive Professional Services from Enfuse.IO
Enfuse.IO is a trusted leader in providing comprehensive data warehouse migration services. They specialize in helping businesses seamlessly transition from legacy data warehouses, such as Greenplum and other enterprise data warehouses (EDWs), to modern platforms. Key services include:

Migration from On-Premise to Cloud:
Expertise in moving data warehouses to leading cloud providers like AWS, Azure, and Google Cloud.
Customized strategies to minimize downtime and ensure data integrity.
On-Premise-to-On-Premise Migration:
Tailored solutions for organizations preferring to retain their data infrastructure on-premises.
Streamlined migration processes to optimize performance and reduce operational disruption.
Comprehensive Assessment and Planning:
Detailed analysis of existing data architecture.
Cost and performance optimization for the target environment.
Post-Migration Support:
Ongoing maintenance and optimization services.
Robust support to ensure peak performance of your data warehouse.
With their proven expertise, Enfuse.IO makes the transition to modern data warehousing smoother and more efficient, enabling businesses to unlock new opportunities for data analytics and insights.

Licensing and ASF Support

Apache License: 
Apache Cloudberry is distributed under the Apache License, which explicitly disclaims any warranties or guarantees of support.
Community Support:
Volunteers may assist through mailing lists or forums, but assistance is not guaranteed and is provided at their discretion.
Important Considerations

For enterprises seeking reliable support and tailored services, organizations like Enfuse.IO and SynxData.com are ideal partners.
Proprietary features and enhanced support are often offered by these companies. Contact them directly to explore solutions that meet your specific needs.
In Summary

Whether you're using Apache Cloudberry for open-source development or looking for enterprise-level solutions, you can rely on commercial providers like Enfuse.IO for migration services and tailored support. Let them help you modernize your data infrastructure and take full advantage of cutting-edge data analytics capabilities.

We’re thrilled to have you join the Apache Cloudberry community. Your collaboration and innovation will help shape the future of this open-source project. Welcome aboard!

Hope this is helpful. Please feel free to reach out to me on Cloudberry Slack or tushar.pednekar@gmail.com if you would like to discuss anything more or dive deeper on any of the topics related to Cloudberry or Greenplum.

--
[Tushar Pednekar](https://linkedin.com/in/tusharpednekar/)
[Apache Cloudberry](https://cloudberry.apache.org/)
